pywb/static/css/playback.css

```diff
@@ -4,4 +4,8 @@
 
 .next_capture {
   margin: 0 15px 0 15px;
+}
+
+#_wb_frame_top_banner {
+  background-color: #8c1515 !important;
```

Comment:
!important is required to override the setting from `default_banner.css` because it has an !important tag.